src/sql/azdata.proposed.d.ts

```diff
@@ -321,5 +322,17 @@ declare module 'azdata' {
 		 */
 		tabs: DashboardTab[];
 	}
+
+	export interface ModelViewDashboardOptions {
+		/**
+		 * Whether to show the tab icon, default is true
+		 */
+		showIcon?: boolean;
+
+		/**
+		 * Whether to show the tab header when there is only one tab, default is false
+		 */
+		showTabsWhenOne?: boolean;
```

Comment:
hmm this name is a little awkward. can you work with PM?

Reply:
What if you just made it "alwaysShowNavigationPane" (default false) - then users would just set that to true if they wanted to show it even when 1 tab is shown

Reply:
And also - I don't really consider the tabs in the navigation pane to be "headers" so I'd consider rewording that.

Reply:
i am renaming it to alwaysShowTabs.